specs/experimental/standard-l2-genesis.md

```diff
@@ -160,15 +178,15 @@ The following fields are included:
 - `to` is `Predeploys.L1Block`
 - `version` is `uint256(0)`
 - `opaqueData` is the tightly packed transaction data where `mint` is `0`, `value` is `0`, the `gasLimit`
-   is `200_000`, `isCreation` is `false` and the `data` is `abi.encodeCall(L1Block.setConfig, (_type, _value))`
+  is `200_000`, `isCreation` is `false` and the `data` is `abi.encodeCall(L1Block.setConfig, (_type, _value))`
 
 #### `upgrade`
 
 The `upgrade` function MUST only be callable by the `UPGRADER` role as defined
 in the [`SuperchainConfig`](#superchainconfig).
 
 ```solidity
-function upgrade(bytes memory _data) external
+function upgrade(uint32 _gasLimit, bytes memory _data) external
```

Comment:
How will the _gasLimit value be set? Will it be hardcoded in the SystemConfig?

Reply:
Also, is `upgrade()` the right name for this? If "`to` is `Predeploys.L1Block`", them I think something like `updateConfig()` might be more appropriate.